nurseconnect/tasks.py

```diff
@@ -53,8 +53,8 @@ def nurses_registered():
         "dataValues": [
             {
                 "dataElement": settings.JEMBI["num_nurses"]["dataElement"],
-                "period":
-                    str(datetime.now().year) + "%02d" % datetime.now().month,
+                "period": "{}{:02d}".format(str(datetime.now().year),
+                                            datetime.now().month),
```

Comment:
I'm thinking we could maybe use `strftime` here instead? (and everywhere else that we're doing this formatting).

We could maybe also store the formatting string for this somewhere central, and then everywhere uses the same formatting string, so that there's one place to change this date format.